### PR TITLE
Add missing test descriptions

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Trial by fire for @zouhir! 🔥 

Only joking, small one to see if you are receiving the requests for review.

File: payment-request/payment-request-abort-method.https.html [hunk 1]
```diff
@@ -70,7 +70,7 @@
   // The request is now "closed", so...
   await promise_rejects(t, "InvalidStateError", request.abort());
   await promise_rejects(t, "AbortError", acceptPromise);
-});
+}, "aborting a request before it is shown doesn't prevent it from being shown later.");
```
Comment (zouhir) [inline payment-request/payment-request-abort-method.https.html]: `aborting` is lowercase while every other description in the file started with uppercase.

heh, joking, you don't have to change it just messing with you for my first review request 😝

**Review by zouhir — APPROVED** [review]

**Comment by zouhir** [comment]
heh @marcoscaceres, request received.

LGTM 🔥

File: payment-request/payment-request-abort-method.https.html (hunk 1 above)
Comment (marcoscaceres) [inline payment-request/payment-request-abort-method.https.html]:
```suggestion
}, "Aborting a request before it is shown doesn't prevent it from being shown later.");
```

Comment (marcoscaceres) [reply payment-request/payment-request-abort-method.https.html]: No, consistency is good. Thanks for pointing that out.

**Review by romandev — APPROVED** [review]
lgtm

**Comment by marcoscaceres** [comment]
Thanks @romandev for the additional check!

**Pull request merged**